torch/quantization/observer.py

```diff
@@ -232,6 +234,11 @@ def _calculate_qparams(self, min_val, max_val):
                     zero_point = zero_point.new_full(zero_point.size(), (qmin + qmax) // 2)
                 else:
                     zero_point = zero_point.new_full(zero_point.size(), 128)
+        elif self.qscheme == torch.per_channel_affine_float_qparams:
+            scale = (orig_max - orig_min) / float(qmax - qmin)
```

Comment:
ideally this should be `max_val - min_val`, since that's what is actually happening.  The other qschemes are not using observed min and max directly.

Reply:
Right. Maybe I can rename the other usages so I can use `max_val - min_val` directly here